scripts/zones/Lufaise_Meadows/IDs.lua

```diff
@@ -41,6 +42,7 @@ zones[xi.zone.LUFAISE_MEADOWS] =
         SPLINTERSPINE_GRUKJUK = GetFirstID('Splinterspine_Grukjuk'),
         KURREA                = GetFirstID('Kurrea'),
         AMALTHEIA             = GetFirstID('Amaltheia'),
+        BITTER_PAST_MOBS      = GetFirstID('Blackbone_Frazdiz'),
```

Comment:
BLACKBONE_FRAZDIZ

Reply:
To make the different mobs readable, just extract the IDs locally at the top of the quest file